src/tests/test_pyproxy.py

```diff
@@ -369,10 +369,10 @@ def test_pyproxy_get_buffer_type_argument(selenium, array_type):
         if fmt.lower() == "q":
             assert result == [hex(x).replace("0x", "") for x in list(mv.cast(fmt))]
         elif fmt == "f" or fmt == "d":
-            from math import isclose
+            from math import isclose, isnan
 
             for a, b in zip(result, list(mv.cast(fmt))):
-                if a and b:
+                if a and b and not (isnan(a) or isnan(b)):
```

Comment:
This test should probably be improved, the `f` and `d` cases are pretty sloppy...
If they are nan shouldn't both be nan?

Reply:
> If they are nan shouldn't both be nan?

Both should be NaN, but I am getting some strange results in selenium and node:

```py
@pytest.mark.skip_pyproxy_check
def test_temp(selenium):
    selenium.run_js(
        """
        let a = pyodide.runPython("bytes(range(252, 256))")
        let buf = a.getBuffer("f32")
        console.log(buf.data[0])
        """
    )
```

This should print `NaN`, but I get `None` in selenium and node. The playwright prints `NaN` correctly.
It also works correctly in the browsers, so maybe there is a bug in our selenium code.